caf_solution/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     // azuread version driven by the caf module
     random = {
       source  = "hashicorp/random"
-      version = "~> 3.3.2"
+      version = "~> 3.3.0"
```

Comment:
Tested with `3.3.1` works fine.

Reply:
Constraints allow last digit update so will tolerate bugfixes, currently in CI/CD with 3.3.2 as per https://github.com/aztfmod/terraform-azurerm-caf/pull/1185/files